app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,59 @@
+class CommentsController < ApplicationController
+
+	before_action :comment_creator?, :only => [:edit, :update, :destroy]
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)